manim/mobject/svg/tex_mobject.py

```diff
@@ -113,6 +113,37 @@ def construct(self):
             tex.set_color_by_tex('igsta', RED)
             self.add(tex)
 
+Note that :func:`~.set_color_by_tex` colors the entire substring containing the Tex searched for,
+not just the specific symbol or Tex expression searched for. Consider the following example:
+
+.. manim:: IncorrectLaTeXSubstringColoring
+    :save_last_frame:
+
+    class IncorrectLaTeXSubstringColoring(Scene):
+        def construct(self):
+            equation = MathTex(
+                "e^x = x^0 + x^1 + \\frac{1}{2} x^2 + \\frac{1}{6} x^3 + \\cdots + \\frac{1}{n!} x^n + \\cdots"
+                )
+            xs = equation.get_parts_by_tex("x")
+            xs.set_color_by_tex(YELLOW)
+            self.add(equation)
+
+As you can see, this colors the entire equation yellow, contrary to what may be expected. To color only ``x`` yellow, we have to do the following:
+
+.. manim:: CorrectLaTeXSubstringColoring
+    :save_last_frame:
+
+    class CorrectLaTeXSubstringColoring(Scene):
+        def construct(self):
+            equation = MathTex(
+                "e^x = x^0 + x^1 + \\frac{1}{2} x^2 + \\frac{1}{6} x^3 + \\cdots + \\frac{1}{n!} x^n + \\cdots", substrings_to_isolate="x"
+                )
+            xs = equation.get_parts_by_tex("x")
+            xs.set_color(YELLOW)
+            self.add(equation)
+
+By setting ``substring_to_isolate`` to ``x``, we split up the :class:`~.MathTex` into substrings automatically and isolate ``x`` components into individual substrings. :func:`~.set_color_by_tex`: can then be used to achieve the desired result.
```

Comment:
Minor rewording (I don't like sentences starting with functions), and removed a superfluous colon.

```suggestion
By setting ``substring_to_isolate`` to ``x``, we split up the :class:`~.MathTex` into substrings
automatically and isolate ``x`` components into individual substrings. Only then can :meth:`~.set_color_by_tex` be used to achieve the desired result.
```